pkg/services/smtp/smtp_config.go

```diff
@@ -3,10 +3,11 @@ package smtp
 import (
 	"errors"
 	"fmt"
-	"github.com/containrrr/shoutrrr/pkg/util"
 	"net/url"
 	"strconv"
 
+	"github.com/containrrr/shoutrrr/pkg/util"
+
```

Comment:
(**very** nit)
```suggestion
```

Reply:
np! did it as a separate commit though as `gofmt` alphabetised those impots. Not sure why it didn't merge them originally